lcd/boilerplate/lib/lcd.rb

```diff
@@ -0,0 +1,31 @@
+class Lcd
+  NUMBER_HASH = {
+    '0' => " -- \n|  |\n|  |\n    \n|  |\n|  |\n -- ",
+    '1' => "    \n   |\n   |\n    \n   |\n   |\n    ",
+    '2' => " -- \n   |\n   |\n -- \n|   \n|   \n -- ",
+    '3' => " -- \n   |\n   |\n -- \n   |\n   |\n -- ",
+    '4' => "    \n|  |\n|  |\n -- \n   |\n   |\n    ",
+    '5' => " -- \n|   \n|   \n -- \n   |\n   |\n -- ",
+    '6' => " -- \n   |\n   |\n -- \n|  |\n|  |\n -- ",
+    '7' => " -- \n   |\n   |\n    \n   |\n   |\n    ",
+    '8' => " -- \n|  |\n|  |\n -- \n|  |\n|  |\n -- ",
+    '9' => " -- \n|  |\n|  |\n -- \n   |\n   |\n    "
+  }
```

Comment:
Would be nice if you could fine a way to represent these strings with real new lines rather than with the "\n" character. Reason I say this is, my head hurts when looking the strings and trying to work out how they will render. Getting the code closer to the output would make it easier to reason about.